### PR TITLE
#115: Totals Filter

### PR DESCRIPTION
**Pull request by ShashankKasera**

### CODE REVIEW

**Comment by ShashankKasera** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Pull request contains heavy changes. Please ensure that your PR is small for faster review.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>Having a proper description on PR gives more clarity to the reviewer. Please ensure that the PR has a neatly written description explaining the purpose of this change request.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Let's also assign this pull request to someone for review.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems">Danger Kotlin</a> against fa7917ffb1e06e17445d53881a2fd79bce2e9352
</p>

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/activity/allexpenses/AllExpensesAdapter.kt [hunk 1]
```diff
@@ -38,6 +39,7 @@ class AllExpensesAdapter(
             tvName.text = expenseItem?.person?.name
             tvAmount.text = expenseItem?.expense?.amount.toString()
             tvDate.text = expenseItem?.expense?.date
+//            tvDate.text = expenseItem?.expense?.date?.dateToString((expenseItem.expense.date),"dd/mm/yyyy")
```
Comment (mayankkasera) [inline app/src/main/java/com/shashank/splitterexpensemanager/feature/activity/allexpenses/AllExpensesAdapter.kt:42]: remove this

**Comment by mayankkasera** [comment]
Remove all unused comments

File: core/src/main/java/com/shashank/splitterexpensemanager/core/extension/String.kt [hunk 2]
```diff
@@ -19,4 +19,6 @@ fun String.capitalizeFirstLetter(input: String): String {
     } else {
         input
     }
-}
\ No newline at end of file
+}
+
+
```
Comment (mayankkasera) [inline core/src/main/java/com/shashank/splitterexpensemanager/core/extension/String.kt:24]: remove these extra line.

Comment (ShashankKasera) [reply core/src/main/java/com/shashank/splitterexpensemanager/core/extension/String.kt:24]: done

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/activity/allexpenses/AllExpensesAdapter.kt (hunk 1 above)
Comment (ShashankKasera) [reply app/src/main/java/com/shashank/splitterexpensemanager/feature/activity/allexpenses/AllExpensesAdapter.kt]: done

**Review by mayankkasera — APPROVED** [review]

**Pull request merged**